test/ejabberd_sm_SUITE.erl

```diff
@@ -79,13 +79,13 @@ init_per_group(cets, Config) ->
     {ok, Pid} = cets_discovery:start(DiscoOpts),
     [{backend, ejabberd_sm_cets}, {cets_disco_pid, Pid} | Config].
 
-init_redis_group(true, Config) ->
+init_redis_group({true, ConnType}, Config) ->
     Self = self(),
     proc_lib:spawn(fun() ->
                   register(test_helper, self()),
                   mongoose_wpool:ensure_started(),
                   % This would be started via outgoing_pools in normal case
-                  Pool = default_config([outgoing_pools, redis, default]),
+                  Pool = redis_pool_config(ConnType),
```

Comment:
We have a pretty useful [helper](https://github.com/esl/MongooseIM/blob/master/test/common/config_parser_helper.erl#L1370) already:
```suggestion
                  Pool = config([outgoing_pools, redis, default], redis_pool_config(ConnType)),
```